beacon.yaml

```diff
@@ -1621,30 +1621,30 @@ components:
       example: GRCh38
     ReferenceBases:
       description: |
-        Reference bases for this variant (starting from `start`). 
-        Accepted values: [ACGTN]*. N is a wildcard, that denotes the 
-        position of any base, and can be used as a standalone base of any 
-        type or within a partially known sequence. For example a sequence 
-        where the first and last bases are known, but the middle portion can 
-        exhibit countless variations of [ACGT], or the bases are unknown: 
-        ANNT the Ns can take take any form of [ACGT], which makes both ACCT 
+        Reference bases for this variant (starting from `start`).
+        Accepted values: [ACGTN]*. N is a wildcard, that denotes the
+        position of any base, and can be used as a standalone base of any
+        type or within a partially known sequence. For example a sequence
+        where the first and last bases are known, but the middle portion can
+        exhibit countless variations of [ACGT], or the bases are unknown:
+        ANNT the Ns can take take any form of [ACGT], which makes both ACCT
         and ATGT (or any other combination) viable sequences.
       type: string
-      pattern: '^([ACGTN]+)$'
+      pattern: '^([ACGTN]*)$'
     AlternateBases:
       description: |
-        The bases that appear instead of the reference bases. Accepted 
-        values: [ACGTN]*. N is a wildcard, that denotes the position of any 
-        base, and can be used as a standalone base of any type or within a 
-        partially known sequence. For example a sequence where the first and 
-        last bases are known, but the middle portion can exhibit countless 
-        variations of [ACGT], or the bases are unknown: ANNT the Ns can take 
-        take any form of [ACGT], which makes both ACCT and ATGT (or any 
+        The bases that appear instead of the reference bases. Accepted
+        values: [ACGTN]*. N is a wildcard, that denotes the position of any
+        base, and can be used as a standalone base of any type or within a
+        partially known sequence. For example a sequence where the first and
+        last bases are known, but the middle portion can exhibit countless
+        variations of [ACGT], or the bases are unknown: ANNT the Ns can take
+        take any form of [ACGT], which makes both ACCT and ATGT (or any
         other combination) viable sequences.
-        
-        Categorical variant queries, e.g. such *not* being represented through 
+
+        Categorical variant queries, e.g. such *not* being represented through
         sequence & position,  make use of the `variantType` parameter.
-        
+
         Optional: either `alternateBases` or `variantType` is required.
       type: string
       pattern: '^([ACGTN]+)$'
```

Comment:
@mbaudis , could you apply the change here too, so we can accept & merge this PR? Thanks!

Reply:
@sdelatorrep Added in https://github.com/ga4gh-beacon/specification-v2/pull/41/commits/14c5a725fb8d3dc6c1e7838803c3d18c0281ff7e with some cleanup of the descriptions